cmd/dimension.go

```diff
@@ -1,27 +1,21 @@
 package cmd
 
 import (
-	"fmt"
-	"os"
-
 	"github.com/qlik-oss/corectl/internal"
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
 )
 
 var setDimensionsCmd = withLocalFlags(&cobra.Command{
-	Use:   "set <glob-pattern-path-to-dimensions-files.json>",
-	Short: "Set or update the dimensions in the current app",
-	Long:  "Set or update the dimensions in the current app",
-	Example: `corectl dimension set
-corectl dimension set ./my-dimensions-glob-path.json`,
+	Use:     "set <glob-pattern-path-to-dimensions-files.json>",
+	Args:    cobra.ExactArgs(1),
+	Short:   "Set or update the dimensions in the current app",
+	Long:    "Set or update the dimensions in the current app",
+	Example: `corectl dimension set ./my-dimensions-glob-path.json`,
```

Comment:
I  guess all these can use "" quotes now since we are no longer supplying multiple lines